src/client/address_book/storage.h

```diff
@@ -54,6 +54,22 @@ namespace client {
 /// @class AddressBookDefaults
 /// @brief Default string constants used throughout address book
 struct AddressBookDefaults {
+  /// @enum Subscription
+  /// @brief Subscription type for where to load/save subscription addresses
+  /// @notes Scoped to prevent namespace pollution (otherwise, purely stylistic)
+  enum struct Subscription
```

Comment:
I'd prefer `SubscriptionType` or something that makes this more obvious of not being an object.

Reply:
Agreed, I'll update the name.